recipe/recipe.yaml

```diff
@@ -29,10 +34,18 @@ requirements:
         - m2-findutils
         - m2-sed
   host:
-    - dotnet
+    - if: not unix
+      then:
+        - dotnet >=8.0.302
+      else:
+        - dotnet >=8.0.302
```

Comment:
Do we also need `dotnet` in host?  That would only make sense if we linking to a shared library provided by `dotnet` or including a header.  Otherwise I think we just want it in build.